imap_processing/codice/packet_definitions/P_COD_LO_NSW_SPECIES_COUNTS.xml

```diff
@@ -33,6 +33,9 @@
 			<xtce:IntegerParameterType name="UINT20" signed="false">
 				<xtce:IntegerDataEncoding sizeInBits="20" encoding="unsigned" />
 			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="UINT23" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="23" encoding="unsigned" />
+			</xtce:IntegerParameterType>
 			<xtce:IntegerParameterType name="UINT24" signed="false">
```

Comment:
I haven't looked closely, but is `UINT24` unused now and should be removed?

Reply:
Yes, thanks!